commands.go

```diff
@@ -152,10 +227,19 @@ func (cmd Command) Run(label string, cmdline ...string) error {
 	}
 
 	// Disable services start/stop for commands running in chroot
+	// Save the original resolv.conf and copy version from host
 	if cmd.ChrootMethod != CHROOT_METHOD_NONE {
 		services := ServiceHelper{cmd.Chroot}
 		services.Deny()
 		defer services.Allow()
+
+	}
+
+	// We use own resolv.conf handling
+	if sum, err := cmd.saveResolvConf(); err != nil {
+		return err
+	} else {
+		defer cmd.restoreResolvConf(sum)
```

Comment:
This will ignore errors when restoring the resolv.conf;  I don't think that's wanted.